src/main/java/com/googlecode/protobuf/format/JsonFormat.java

```diff
@@ -875,6 +885,36 @@ protected void mergeField(Tokenizer tokenizer,
             mergeField(tokenizer, extensionRegistry, builder);
         }
     }
+    
+    private void handleUnknownField(Tokenizer tokenizer,    										
+    		Message.Builder builder,
+    		int fieldNumber) throws ParseException {    	
+    	tokenizer.tryConsume(":");
+    	if ("{".equals(tokenizer.currentToken())) {

```

Comment:
If I understand correctly, tryConsume "tries" to consume the character you pass and returns a result - you can use it instead of
```
if ("something".equals(tokenizer.currentToken())) {
    tokenizer.consume("something");
}
```